src/components/Hero/index.tsx

```diff
@@ -34,8 +34,8 @@ const Hero = ({ title, subTitle }: Props): JSX.Element => {
     <div className="home-page-hero">
       <h1>{title}</h1>
       <h2 className="sub-title t-subheading">{subTitle}</h2>
-      <div className="btn-ctas">
-        <div className="download-lts-container">
+      <div className="btn-ctas" style={{textTransform:'none'}} >
```

Comment:
None of the code in this repo uses inline styles.
Styling for Hero should be done in Hero.scss